### DISCUSSION TITLE
mass post editor improvements

### DISCUSSION THREAD

**Discussion by nuvema572** [AprilSylph/XKit-Rewritten#294] [Suggestion Box]
i use the mass post editor a LOT, so it would be awesome if the improvements from new xkit were transferred over. it would also be really helpful to have an option to see what posts _aren't_ tagged as a certain tag, if possible.

**Comment by AprilSylph** [comment]
The mass post editor is still using Tumblr's legacy code, which XKit Rewritten will not attempt to tackle. Continue using New XKit for features like this; the two addons should not inherently conflict.

**Comment by nuvema572** [comment]
thanks for the quick reply! i figured it used the legacy code but couldn't be sure. thank you!
